polaris.shopify.com/content/contributing/index.md

```diff
@@ -17,11 +17,23 @@ Contribution to Polaris looks different depending on whether you work at Shopify
+| **🤝 Collaborate with other disciplines** <br>A good contribution addresses the needs and perspectives of design, content, and code. Consult a diverse group of disciplines when drafting your contribution.                                                                                                       | **🪟 Work in the open** <br>Public discussion is crucial for contribution visibility and feedback. Submit contributions using [GitHub](https://github.com/Shopify/polaris/discussions/6750). Shopify employees can request assistance via the [#polaris Slack channel](https://shopify.slack.com/archives/C4Y8N30KD).                                                                                                                                    |
+| **📅 Include contribution in project cycle** <br>Contributing close to a project deadline creates blockers and substandard work. Include contribution in project retros, or involve Polaris early in a project cycle. Read [Working with Polaris]().                                                               | **🔍 Check your work** <br>Double check your contribution to make sure it meets standards for things like accessibility and performance. Following the guidance for each contribution type is also a great way to meet quality standards.                                                                                                                                                                                                                |
+| **🏗️ Consider the impact on the system** <br>Contributions to Polaris should apply to multiple admin surfaces. They should not be unique solutions with only one use case. Read [Designing with a system](/contributing/designing-with-a-system), and reach out to the Polaris team for system advice or guidance. | **🔬 Be clear and detailed** <br>Submit a short summary of your contribution, along with relevant links, artifacts, and context. State your expectations, understanding that we must prioritize certain business and system needs. If you can ship in collaboration with another product team, tag them and let the Polaris team know. Capture contribution details with [GitHub issue templates](https://github.com/Shopify/polaris/issues/new/choose). |
+|                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Love this. What about just using a level three header for `Preparing contributions` and put the content below it. Tables are not the best unless they have tabular data.

Reply:
@alex-page Yeah, it's not ideal but we felt without the table giving us the artificial 2-column layout it bloats the page to be far too long. This at least helps it be more scannable.

Reply:
Awesome. Thanks for the context 🚀